components/CostsPie.client.vue

```diff
@@ -218,7 +218,7 @@ const populateCostsDataIntoPie = () => {
     costsData.push({
       id: cost.id,
       parent: appStore.totalCost!.id,
-      name: appStore.getCostLabel(cost.id),
+      name: `${appStore.getCostLabel(cost.id)}${cost.id === "life_years" ? "*" : ""}`,
```

Comment:
What does this affect - the label in the tooltips?

Reply:
I don't think it looks good in the cost pie labels - sorry! I think it's best to just have the * once, in the table.

![image](https://github.com/user-attachments/assets/976488ab-585d-4a3d-8065-340b0530df0c)